runhouse/resources/hardware/on_demand_cluster.py

```diff
@@ -454,6 +454,19 @@ def _update_from_sky_status(self, dryrun: bool = False):
         cluster_dict = self._sky_status(refresh=not dryrun)
         self._populate_connection_from_status_dict(cluster_dict)
 
+    def _update_from_den_response(self, config: dict):
```

Comment:
i think this should live in the DenLauncher class and be called at the end of the `up` method